lib/index.rb

```diff
@@ -1,20 +1,56 @@
+# !/usr/bin/env ruby -I ../lib -I lib
 # frozen_string_literal: true
-#!/usr/bin/env ruby -I ../lib -I lib
-# coding: utf-8
 
 require 'sinatra'
 require './lib/file_customer_gateway'
 require './lib/use_cases/save_customer_details'
+require './lib/use_cases/view_summary'
 require './lib/builder/customer'
 require './lib/domain/address'
 require './lib/domain/customer'
 
+class ItemsGateway
+  def all
+    [
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
+      { part_id: 454, part_name: 'Uranium', part_price: 130, part_quantity: 100 },
+      { part_id: 767, part_name: 'ACHSDJVHJDWVVFWVYEUVFW', part_price: 10, part_quantity: 200000 },
+      { part_id: 999, part_name: 'Screws', part_price: 0.9, part_quantity: 2000 },
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
+      { part_id: 454, part_name: 'Uranium', part_price: 130, part_quantity: 100 },
+      { part_id: 767, part_name: 'ACHSDJVHJDWVVFWVYEUVFW', part_price: 10, part_quantity: 200000 },
+      { part_id: 999, part_name: 'Screws', part_price: 0.9, part_quantity: 2000 },
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
+      { part_id: 454, part_name: 'Uranium', part_price: 130, part_quantity: 100 },
+      { part_id: 767, part_name: 'ACHSDJVHJDWVVFWVYEUVFW', part_price: 10, part_quantity: 200000 },
+      { part_id: 999, part_name: 'Screws', part_price: 0.9, part_quantity: 2000 },
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
+      { part_id: 454, part_name: 'Uranium', part_price: 130, part_quantity: 100 },
+      { part_id: 767, part_name: 'ACHSDJVHJDWVVFWVYEUVFW', part_price: 10, part_quantity: 200000 },
+      { part_id: 999, part_name: 'Screws', part_price: 0.9, part_quantity: 2000 },
+      { part_id: 456, part_name: 'Drones', part_price: 144, part_quantity: 34 },
+      { part_id: 454, part_name: 'Uranium', part_price: 130, part_quantity: 100 },
+      { part_id: 767, part_name: 'ACHSDJVHJDWVVFWVYEUVFW', part_price: 10, part_quantity: 200000 },
+      { part_id: 999, part_name: 'Screws', part_price: 0.9, part_quantity: 2000 }
+    ]
+  end
+end
+
 get '/customer-details' do
   @customer_details = {}
   @errors = []
   erb :customer_details
 end
 
+get '/order-summary' do
+  file_items_gateway = ItemsGateway.new
```

Comment:
We renamed our class FileItemsGateway to match the customer gateway. You'll need to rename this once we've all merged.